### ISSUE TITLE
Style bug: Charts - space around circle nodes missing

### ISSUE THREAD

**Issue by orion-cengage** [cengage/react-magma#1545]
**Describe the bug**
A styling issue was introduced in 4.5.0 where the clear space around the circle nodes on line charts is either missing or only partial, which is going to cause us a11y issues.

Screenshot of styling bug
![image](https://github.com/user-attachments/assets/dbf20578-bb45-48a1-a807-80270c7f2130)
![image](https://github.com/user-attachments/assets/2d63d9a7-f287-45e1-bad0-856ee781dcb9)

**To Reproduce**
1. Go to Storybook or Docs (4.5 or 4.6) and look at any of the charts based on lines, such as Line and Area
2. Notice that the space around the circles that gives them visual separation is only kind of visible on the right side
3. Also notice that if you hover over a circle node, it fixes itself.
4. To see the correct styling you can go to Docs 4.4

**Expected behavior**
![image](https://github.com/user-attachments/assets/59016d8c-983b-4520-8afe-dda7de2c3e00)
![image](https://github.com/user-attachments/assets/847a4218-a4e7-4739-853f-f90009ae31c2)

**Comment by chris-cedrone-cengage** [comment]
There isn't a white border on the circles with Safari 18

<img width="1198" alt="Safari" src="https://github.com/user-attachments/assets/40f4f95a-65bd-44c1-9cb6-0c61a01c655c">

**Comment by orion-cengage** [comment]
I tested as well, and the only issue I found was the same Safari issue Chris pointed out.

**Comment by nikitaorliak-cengage** [comment]
In our case, Safari doesn't support `filter: drop-shadow`, so we decided to leave it as it is until Safari updates/Other solutions.
